dev_guide/managing_images.adoc

```diff
@@ -371,6 +371,13 @@ By default, all service accounts in a project have rights to pull any image in
 the same project, and the *builder* service account has rights to push any image
 in the same project.
 
+ifdef::openshift-online[]
+Dev Preview of the Next Generation of OpenShift Online gives developers a
+hands-on preview of the OpenShift platform in a hosted environment that includes
+access to the internal registry. The internal registry is located at
+*_registry.preview.openshift.com_*.
+endif::[]
```

Comment:
Maybe this could be the 2nd paragraph in this section?

Also should probably use the name form: {product-title} (Next Gen) Developer Preview

Maybe: s/the internal registry/an internal registry, located at blah/